specviz/plugins/arithmetic/arithmetic_editor.py

```diff
@@ -245,10 +250,10 @@ def _update_status(self):
                 self.label_status.setStyleSheet('color: red')
                 self.label_status.setText("Incomplete or invalid syntax")
                 self.button_ok.setEnabled(False)
-            except Exception as exc:
-                self.label_status.setStyleSheet('color: red')
-                self.label_status.setText(str(exc))
-                self.button_ok.setEnabled(False)
+            # except Exception as exc:
+            #     self.label_status.setStyleSheet('color: red')
+            #     self.label_status.setText(str(exc))
+            #     self.button_ok.setEnabled(False)
```

Comment:
if these aren't wanted here, probably best to just delete the code.

Reply:
You're right; took them out for debugging and never uncommented. Fixed.